expenses-manager-frontend/src/main/frontend/src/app/daily-stats/daily-stats.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="dashboard-container">
+
+  <div class="date-picker-container">
+    <button mat-icon-button (click)="previousDay()">
+      <mat-icon>chevron_left</mat-icon>
+    </button>
+
+    <button mat-button (click)="picker.open()">
+      {{ currentDate | date: 'EEEE - d MMMM y' }}
+    </button>
+
+    <button mat-icon-button (click)="nextDay()">
+      <mat-icon>chevron_right</mat-icon>
+    </button>
+
+    <input matInput [matDatepicker]="picker" (dateChange)="onDateChange($event)" [max]="maxDate" style="display: none;">
+    <mat-datepicker #picker></mat-datepicker>
+  </div>
+
+  <div class="chart-container">
+    <ngx-charts-pie-chart
+      [view]="[300, 300]"
+      [results]="data"
+      [legend]="false"
+      [doughnut]="true"
+      [labels]="false"
+      [scheme]="colorScheme">
+    </ngx-charts-pie-chart>
+
+    <div class="legend-container">
+      <ul>
+        <li *ngFor="let category of categories">
```

Comment:
Aici nu as lista toate categoriile existente in baza de date, ci doar cele care sunt in chart pe ziua respectiva. Iar daca nu e nicio cheltuiala pe ziua selectata sa afisam un mesaj de genul "No expenses found! Please select another day"